src/grt/src/GlobalRouter.cpp

```diff
@@ -1067,6 +1067,9 @@ void GlobalRouter::computeUserLayerAdjustments(int max_routing_layer)
                 = fastroute_->getEdgeCapacity(x - 1, y - 1, x, y - 1, layer);
             int new_h_capacity
                 = std::floor((float) edge_cap * (1 - adjustment));
+            new_h_capacity = edge_cap && adjustment != 1 > 0
```

Comment:
adjustment != 1 > 0  seem strange - what are you trying to express?

Reply:
It was an error from my end. Fixed in the latest commit.